.gemini/styleguide.md

```diff
@@ -21,3 +21,13 @@ auto-formatter, for each language, as described in
 - PR descriptions should include the Pre-Review Checklist from
   [the PR template](https://github.com/flutter/packages/blob/main/.github/PULL_REQUEST_TEMPLATE.md),
   with all of the steps completed.
+
+## Review Agent Guidelines
+
+When providing a summary, the review agent must adhere to the following principles:
+- **Be Objective:** Focus on a neutral, descriptive summary of the changes. Avoid subjective value judgments
+  like "good," "bad," "positive," or "negative." The goal is to report what the code does, not to evaluate it.
```

Comment:
The "not to evaluate it" may end up swinging too far the other way; we'll have to see if it stops pointing out high-level issues (e.g., insufficient testing).